atc/api/accessor/role_action_map.go

```diff
@@ -75,6 +75,7 @@ var requiredRoles = map[string]string{
 	atc.ListWorkers:                   "viewer",
 	atc.DeleteWorker:                  "member",
 	atc.SetLogLevel:                   "member",
+	atc.ListActiveUsersSince:          "member",
```

Comment:
I may be wrong, but I think this is supposed to be for admins only?

Reply:
I also might be wrong haha but I thought the admin privileges were checked by another handler https://github.com/concourse/concourse/blob/e3a5020891b8f0858b31cf4f504a6789a0662c19/atc/wrappa/api_auth_wrappa.go#L116-L122

Reply:
@pivotal-jamie-klassen Ok I think you're right, I talked to Josh and he said that it doesn't make sense for these endpoints to even have a "team role" because it is more of a concourse-wide role as admin.

I think I'm going to change my test to ignore any that requires admin privileges, thanks a lot for your help!